index.html

```diff
@@ -3131,12 +3147,84 @@ <h2>
       </pre>
       <p class="note">
         A <a>PaymentResponse</a> is returned when a user has selected a payment
         method and approved a payment request.
       </p>
+      <section>
+        <h2>
+          <dfn>retry()</dfn> method
+        </h2>
+        <p data-tests="payment-response/retry-method-manual.https.html">
+          The <a>retry()</a> method MUST act as follows:
+        </p>
+        <ol class="algorithm">
+          <li>Let <var>response</var> be the <a>context object</a>.
+          </li>
+          <li>Let <var>request</var> be
+          <var>response</var>.<a>[[\request]]</a>.
+          </li>
+          <li>If <var>response</var>.<a>[[\complete]]</a> is true, return <a>a
+          promise rejected with</a> an "<a>InvalidStateError</a>"
+          <a>DOMException</a>.
+          </li>
+          <li>If <var>response</var>.<a>[[\retrying]]</a> is true, return <a>a
+          promise rejected with</a> an "<a>InvalidStateError</a>"
+          <a>DOMException</a>.
+          </li>
+          <li>Set <var>response</var>.<a>[[\retrying]]</a> to true.
+          </li>
+          <li>Set <var>request</var>.<a>[[\state]]</a> to "<a>interactive</a>".
+          </li>
+          <li>Let <var>retryPromise</var> be <a>a new promise</a>.
+          </li>
+          <li>Set <var>response</var>.<a>[[\retryPromise]]</a> to
+          <var>retryPromise</var>.
+          </li>
+          <li>
+            <a>In parallel</a>:
+            <ol>
+              <li>In the payments UI, indicate to the end-user that something
+              is wrong with the user-provided data of the payment response.
+              </li>
+              <li>
+                <p>
+                  The <var>retryPromise</var> will later be resolved or
+                  rejected by either the <a>user accepts the payment request
+                  algorithm</a> or the <a>user aborts the payment request
+                  algorithm</a>, which are triggered through interaction with
+                  the user interface.
+                </p>
+                <p data-test="rejects_if_not_active.https.html">
+                  If <var>document</var> stops being <a data-cite=
+                  "!HTML#fully-active">fully active</a> while the user
+                  interface is being shown, or no longer is by the time this
+                  step is reached, then:
+                </p>
+                <ol>
+                  <li>Close down the user interface.
+                  </li>
+                  <li>Set the <a>user agent</a>'s <a>payment request is
+                  showing</a> boolean to false.
+                  </li>
+                  <li>Reject <var>retryPromise</var> with an
+                  "<a>AbortError</a>" <a>DOMException</a>.
+                  </li>
+                </ol>
+                <p>
+                  Finally, when <var>retryPromise</var> settles, set
+                  <var>response</var><a>[[\retrying]]</a> to false.
```

Comment:
There are a lot of missing periods in this PR. Search for `</var><a>[[` (should be `</var>.<a>[[`).